doc/thingpedia-tutorial-website-monitor.md

```diff
@@ -12,7 +12,7 @@ about the device:
 
 - ID: `<your-name>.edu.stanford.cs294s` (Each device in Thingpedia needs an unique ID, so use your name or email address
 in the device name to make sure it won't conflict with others)
-- Name: `My CS294S`
+- Name: `My CS294S` (Or any name you want!)
```

Comment:
We should probably change the class number... Or pick a website that is not a class website!

Reply:
Ehhhhh I'll leave it for now, I'll try to find an appropriate website if I have time. Trying to debug almond-voice.